src/components/ControlledNumberInputField/ControlledNumberInputField.js

```diff
@@ -0,0 +1,53 @@
+import { makeStyles, TextField } from "@material-ui/core"
+import { Controller } from "react-hook-form"
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+    fontSize: theme.typography.fontSize,
+    fontFamily: theme.typography.fontFamily,
+
+    "& .MuiFormHelperText-contained": {
+      margin: "0",
+    },
+
+    "& .MuiOutlinedInput-input": {
+      padding: "6px 10px 6px 10px",
+      fontSize: theme.typography.fontSize,
+    },
+  },
+  label: {
+    fontWeight: theme.typography.fontWeightBold,
+    paddingBottom: "4px",
+  },
+}))
+
+const ControlledNumberInputField = ({ control, name, label, placeholder }) => {
+  const classes = useStyles()
+
+  return (
+    <div className={classes.root}>
+      <span className={classes.label}>{label}</span>
+
+      <Controller
+        control={control}
+        name={name}
+        render={({ value, onChange }) => {
+          return (
+            <TextField
+              value={value}
+              placeholder={placeholder}
+              variant="outlined"
+              onChange={(event) => onChange(event.target.value)}
+              size="small"
+              type="number"
+            />
```

Comment:
Can you change your number input to one that works like this:
![image](https://user-images.githubusercontent.com/23342131/231640297-85ae5ee7-f0e7-4cde-9440-c5c23e754daa.png)


So make sure these arrows on the side are not shown, and the user must key in the number they want
![image](https://user-images.githubusercontent.com/23342131/231640458-1eb2e850-348f-403d-bf9a-1fff828d7ef1.png)